lib/bolt/outputter.rb

```diff
@@ -27,6 +27,10 @@ def indent(indent, string)
       string.gsub(/^/, indent.to_s)
     end
 
+    def wrap(string, width = 80)
```

Comment:
This probably just needs to be in the human outputter, since we shouldn't wrap JSON output or logs.

Reply:
Whoops, that shouldn't be there at all. Removed.

Reply:
Uhh, I still see it?

Reply:
I might have misinterpreted this comment https://github.com/puppetlabs/bolt/pull/2203#discussion_r493065255 and put it back. I'll move it to `Bolt::Outputter::Human`.